acceptance/single_key_test.go

```diff
@@ -95,7 +95,7 @@ func TestSingleKey(t *testing.T) {
 					if err := v.UnmarshalBinary(r.ValueBytes()); err != nil {
 						return err
 					}
-					return txn.Commit(txn.B.Put(key, v+1))
+					return txn.Commit(client.B().Put(key, v+1))
```

Comment:
Unfortunately, a `Batch()` method conflicts with the `Batch` type. I didn't like `{DB,Txn}.Batch()` because it felt confusing that the created `Batch` was independent of the `{DB,Txn}`. Perhaps `client.NewBatch()` and a `client.B()` alias. Or just skip the alias and be explicit. There are not that many uses of `client.B()`.